reuse.go

```diff
@@ -46,6 +75,53 @@ func newReuse() (*reuse, error) {
 	}, nil
 }
 
+func (r *reuse) runGarbageCollector() {
+	ticker := time.NewTicker(garbageCollectInterval)
+	defer ticker.Stop()
+
+	for range ticker.C {
```

Comment:
uber nit: `for now := range ticker.C {`. We can skip the second call to `now`.

Reply:
Not an uber nit, this saves one syscall per tick.

Reply:
Fixed.

Reply:
Depends on the platform. Go will use vDSO on Linux.